packages/api/cms-api/src/auth/strategies/static-credentials-basic.strategy.ts

```diff
@@ -3,21 +3,26 @@ import { PassportStrategy, Type } from "@nestjs/passport";
 import { BasicStrategy } from "passport-http";
 
 interface StaticCredentialsBasicStrategyConfig {
-    username: string;
+    username?: string;
     password: string;
+    strategyName?: string;
 }
 
-export function createStaticCredentialsBasicStrategy(config: StaticCredentialsBasicStrategyConfig): Type {
+export function createStaticCredentialsBasicStrategy({
+    username: requiredUsername = "comet",
+    password: requiredPassword,
+    strategyName = "static-credentials-basic",
```

Comment:
I would make this parameter non-optional. And in the typical example (that uses badly named API_PASSWORD) it could be named `system-user` or `system-crendentials` or `system`.